src/table.js

```diff
@@ -1341,6 +1341,7 @@ Table.prototype.sampleRowKeysStream = function(gaxOptions) {
 };
 
 /**
+<<<<<<< HEAD
```

Comment:
This looks like a bad merge.  Please delete `<<<<<<< HEAD`

Reply:
fixed